### PR TITLE
Rest crud

### PR DESCRIPTION
**Pull request by mdanter**

### CODE REVIEW

File: ansible-stack-swagger-.0.2.0.yaml [hunk 1]
```diff
@@ -6,7 +6,53 @@ info:
...
+        - in: body
+          name: body
+          description: Customer object that needs to be added to the store
+          required: false
+          schema:
+            $ref: "#/definitions/Customer"
+      responses:
+        "405":
+          description: Invalid input
+    put:
```
Comment (sherl0cks) [inline ansible-stack-swagger-.0.2.0.yaml]: Shouldn't PUT be on `customers\{ID}`? This idea applies to all the resources

**Comment by sherl0cks** [comment]
@mdanter looks good other than the one comment I made inline. Please update and then we can merge.

**Comment by mdanter** [comment]
@sherl0cks take a look now, made the changes you suggested, thanks!

**Comment by sherl0cks** [comment]
thanks. lgtm

**Pull request merged**